src/webdriverio-tester.js

```diff
@@ -0,0 +1,192 @@
+/**
+ * @author Adam Meadows [@job13er](https://github.com/job13er)
+ * @copyright 2015 Cyan, Inc. All rights reserved
+ */
+
+'use strict';
+
+/**
+ * @typedef Result
+ * @property {Number} code - the exit code of the command
+ * @property {String} stdout - the standard output from command
+ */
+
+var _ = require('lodash');
+var path = require('path');
+var sh = require('execSync');
```

Comment:
Not sure how you feel about this, but my gut tells me that using the `...Sync()` functions in node is cheating and that we should be doing things the "right" way by making all calls synchronous. Probably makes no difference at all for a little shell-scripty function like this, but Q does make this quite a bit easier, if you're interested:

```
var sh = Q.nfbind(require('exec');

sh('curl -s -O ' + server + '/' + filename).done(function (text) {
   // whatever
});
```